### PR TITLE
Disable static libraries

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/conda-forge/blosc-feedstock/issues/54

Policy in conda-forge is not to include static libraries. These can be broken out in a separate split package on request.

https://github.com/conda-forge/cfep/blob/main/cfep-18.md

cc @hmaarrfk

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
Rerender locally?

**Comment by jakirkham** [comment]
Yeah org forks are fun. Will try re-rendering locally. Though not expecting much change from PR ( https://github.com/conda-forge/blosc-feedstock/pull/53 )

**Comment by hmaarrfk** [comment]
also tests need to be updated. I can't edit your fork either.

File: .ci_support/linux_64_.yaml [hunk 1]
```diff
@@ -14,7 +14,11 @@ cxx_compiler_version:
 - '10'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+lz4_c:
+- 1.9.3
 pin_run_as_build:
+  lz4-c:
+    max_pin: x.x.x
```
Comment (jakirkham) [inline .ci_support/linux_64_.yaml:21]: Interesting. Would have thought PR ( https://github.com/conda-forge/blosc-feedstock/pull/53 ) would have gotten this as it did the other dependencies when re-rendering.

**Comment by jakirkham** [comment]
Yeah it's not possible to enable edits on org forks. GH disabled that a while back (use to be possible)

Yep agree tests will need to change. Wanted to see what happens on Windows first (don't have access to a Windows machine)

File: .ci_support/linux_64_.yaml (hunk 1 above)
Comment (hmaarrfk) [reply .ci_support/linux_64_.yaml:21]: We should delete this right: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L266

**Comment by hmaarrfk** [comment]
oh "org" not "old"

**Comment by jakirkham** [comment]
Hopefully that does the trick. Though please let me know if you spot any issues

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

File: .ci_support/linux_64_.yaml (hunk 1 above)
Comment (hmaarrfk) [reply .ci_support/linux_64_.yaml:21]: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2860

**Comment by jakirkham** [comment]
Thanks Mark! 🙏

**Pull request merged**